webui/react/package.json

```diff
@@ -59,15 +60,15 @@
     ]
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^6.1.14",
-    "@storybook/addon-backgrounds": "^6.1.14",
-    "@storybook/addon-docs": "^6.1.14",
-    "@storybook/addon-knobs": "^6.1.14",
-    "@storybook/addon-links": "^6.1.14",
-    "@storybook/addon-viewport": "^6.1.14",
-    "@storybook/addons": "^6.1.14",
-    "@storybook/preset-create-react-app": "^3.1.5",
-    "@storybook/react": "^6.1.14",
+    "@storybook/addon-actions": "^6.2.3",
+    "@storybook/addon-backgrounds": "^6.2.3",
+    "@storybook/addon-docs": "^6.2.3",
+    "@storybook/addon-knobs": "^6.2.3",
+    "@storybook/addon-links": "^6.2.3",
+    "@storybook/addon-viewport": "^6.2.3",
+    "@storybook/addons": "^6.2.3",
+    "@storybook/preset-create-react-app": "^3.1.7",
+    "@storybook/react": "^6.2.3",
```

Comment:
upgraded to remove a lot of Prototype Pollution vulnerabilities listed for `storybook 6.1.14`